x/gravity/keeper/attestation.go

```diff
@@ -36,7 +44,7 @@ func (k Keeper) Attest(
 	// and prevents validators from submitting two claims with the same nonce.
 	// This prevents there being two attestations with the same nonce that get 2/3s of the votes
 	// in the endBlocker.
-	lastEventNonce, err := k.GetLastEventNonceByValidator(ctx, valAddr)
+	lastEventNonce, err := k.GetLastEventNonceByValidator(ctx, sdk.ValAddress(valAddr))
```

Comment:
This is not the correct way to convert bech32 to Hex. Use address codec here too.